pyproject.toml

```diff
@@ -22,18 +22,18 @@ dpath = "^2.1.6"
 appdirs = "^1.4.4"
 
 websocat = "^1.12.0"
-golem-node = {version = "0.15.0a18", source = "test-pypi"}
+golem-node = "0.13.3"
```

Comment:
I'd still insist on using the newest available version of 0.15 in most of our tests, in order to have a chance to find issues with it before we publicly release a ray-on-golem version which utilizes it

Reply:
The problem solved itself as yagna in version 0.15.0 is available publicly.